### PR TITLE
Update Web maps want

### PR DESCRIPTION
**Pull request by prushforth**
Add privacy and accessibility tags.  Update description to be less lame.  cc @nchan0154 to validate  these changes, because she's co-author.

### CODE REVIEW

File: wants/61.md [hunk 1]
```diff
@@ -5,11 +5,13 @@ submitter: Peter Rushforth & Nic Chan
...
+Web maps are often seen as used only for store locator pages.  In this era of climate change and intelligent everything, maps could be so much more!  We think that integrating maps (and location) into HTML could be vital for a more private, more accessible society and world.
 
-What we want is to make it easier to create and use web maps and spatial data by implementing maps in browsers. A native HTML `map` element could improve the map viewing experience on so many fronts, including privacy, accessibility, and progressive enhancement.
+What we want in the Web is to make it easier to create and use web maps and spatial data by implementing maps in browsers. A native HTML `map` (or other, newer) element could improve the map viewing experience on so many fronts, including privacy, accessibility, performance and so on.
 
 Our goal, and the goal of the [W3C Maps4HTML Community Group](https://www.w3.org/community/maps4html/) is to establish a standard method for web developers to include interactive web maps within HTML documents.
```
Comment (Malvoz) [inline wants/61.md:17]:
```suggestion
Our goal, and the goal of the [W3C Maps for HTML Community Group](https://www.w3.org/community/maps4html/) is to establish a standard method for web developers to include interactive web maps within HTML documents.
```

**Review by Malvoz — CHANGES_REQUESTED** [review]

**Review by Malvoz — APPROVED** [review]

**Comment by Malvoz** [comment]
@aarongustafson would you like to merge please?

**Comment by prushforth** [comment]
This would be great to get merged, in light of the [survey](https://github.com/WebWeWant/webwewant.fyi/discussions/108#discussioncomment-2290506). Thanks!

**Review by tantek — COMMENTED** [review]
Most of this update looks good. Would folks be open to some grammar improvements? E.g. the sentence "Web maps are often seen as used only for store locator pages."

**Comment by prushforth** [comment]
@tantek thanks, of course!  I don't know if what I did is what you had in mind, but if you can improve it, please have a shot at it.

**Review by tantek — APPROVED** [review]
Much better! LGTM. Thanks for this update.

**Pull request merged**